build.rs

```diff
@@ -1,35 +1,24 @@
-use std::io::ErrorKind;
-use std::process::*;
+extern crate which;
 
-const PATH_PACKAGES : &'static str = "package.json";
+use std::process::Command;
+use which::which;
+
+const PATH_PACKAGES: &'static str = "package.json";
 
 fn main() {
     // Install Node packages if necessary.
     println!("cargo:rerun-if-changed={}", PATH_PACKAGES);
 
-    // First try with `yarn`, if available.
-    println!("Bootstrap: Attempting to install dependencies with yarn.");
-    match Command::new("yarn")
+    let tool =
+        which("yarn")
```

Comment:
Oh, that's good. I wasn't aware of `which`.

Reply:
Me neither before this :)